_module_templates/module_macros.md

```diff
@@ -0,0 +1,328 @@
+<!--
+
+author:   DART Team
+email:    dart@chop.edu
+version:  1.0.0
+language: en
+narrator: UK English Female
+title: Module Macros
+comment:  This is placeholder module to save macros used in other modules.
+
+@overview
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
```

Comment:
```suggestion
**Estimated time to completion:** @estimated_time_in_minutes minutes
```

to avoid the issue of us writing time estimates in various ways, I'm thinking we include the text for the unit within the larger overview macro, and then update the "estimated_time" YAML field to "estimated_time_in_minutes" and then do validation in the rest of the YAML linting process to ensure that the value is an integer between 0-60.

Reply:
when I last looked (a few months ago, granted) all our modules were under 1 hour, just written in a variety of ways. 

I have memory of <60 mins being a hard rule, but I can't find that anywhere now (not in QA template, e.g., or in how-to.md) I also can't find any evidence of it being a soft rule (that we're supposed to shoot for it, as you said) but the fact that you also remember that makes me feel less uncertain. 

I wonder though, if one option is to restrict to under 60 mins, and then if down the line a module comes around that requires a higher limit, we can just chat about it as a team then, and adjust the hard cap to accommodate whatever we want that higher upper limit to be (perhaps 90, or whatever makes sense in the given context). 

That way, if we ever do deviate from 60 mins, we're doing so as a deliberate decision, and we have documentation of us adjusting that criteria rather than a slow creep in length.

Reply:
That sounds like a good approach to me!